index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MyPortofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+    <header class="navbar container fixed-top">
+        <a href="" id="logo">
+            Welcome
+        </a>
+        <nav class="menu">
+            <i class="fa-solid fa-bars "></i>
+        </nav>
+    </header>
+    <section class="header">
```

Comment:
- [ ] It could be better if you use a class name that is not a key word in `html` 👍  , kindly change that 😄